bundles/admin/admin-layereditor/view/AdminLayerForm/AdminLayerFormHandler.js

```diff
@@ -519,11 +519,11 @@ class UIHandler extends StateHandler {
                 return Promise.reject(new Error('Capabilities fetching failed with status code ' + response.status + ' and text ' + response.statusText));
             }
         }).then(json => {
-            const composingModel = this.mapLayerService.getComposingModelForType(layer.type);
+            const updateLayer = { ...this.getState().layer, version };
```

Comment:
Version is undefined here. Should be params.version?

Reply:
Yeah, fixed.